### PR TITLE
Rework cmd

### PR DESCRIPTION
**Pull request by zweihander**
Just for code readability
No runtime/prompt changes

BUT have two differences between current impl.:
1. different help output
2. show-tables & version flags now are commands (I think it's more correct)

### CODE REVIEW

**Comment by zweihander** [comment]
Expectedly, CI has failed due to the lack of a flag `-v`

**Comment by filhodanuvem** [comment]
@zweihander is that possible to create an alias `-v` that behaves like `gitql version` ? I would like to avoid break compatibilities for a while. 
Besides that, nice work, I believed that we are using https://github.com/spf13/cobra already.

**Comment by zweihander** [comment]
@filhodanuvem done

File: main.go [hunk 1]
```diff
@@ -38,28 +38,26 @@ func main() {
...
-							if i+1 < len(fields) {
-								comma = ", "
-							}
-							fmt.Printf("%s%s", field, comma)
-						}
-						fmt.Println()
-					}
-					return nil
-				},
+				Action:  showTablesCmd,
```
Comment (filhodanuvem) [inline main.go:61]: Nice one!

File: main.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
+package main
```
Comment (filhodanuvem) [inline main.go:1]: happy to see a main.go again 😄

**Review by filhodanuvem — APPROVED** [review]

**Pull request merged**